.gitignore

```diff
@@ -35,13 +35,13 @@ captures/
 
 # Intellij
```

Comment:
hmm, it looks like anything in the .serverless/ file should go in the git ignore as well. the files from .serverless directory look like they're just generated output when you run serverless framework, which we don't need to keep. they may also contain sensitive information or environment information.

Reply:
I have taken care of this.